include/rocksdb/statistics.h

```diff
@@ -431,6 +431,13 @@ enum Tickers : uint32_t {
   NON_LAST_LEVEL_READ_BYTES,
   NON_LAST_LEVEL_READ_COUNT,
 
+  // # of synchronous read calls made
+  // Currently enabled in RocksDB prefetcher
+  NUM_SYNC_READS,
```

Comment:
Since its specific to prefetching, maybe we can change the names to NUM_SYNC_PREFETCH and NUM_ASYNC_PREFETCH.